app/controllers/users/sessions_controller.rb

```diff
@@ -15,23 +15,31 @@ class Users::SessionsController < Devise::SessionsController
 
   # and this
   def create_parse_user_if_needed
-    if ENV['PARSE_ID'] and request.subdomain == ENV['PARSE_SUBDOMAIN']
```

Comment:
This is one thing I don't think you should ever touch. We'll probably remove it soon.. and it was only added for a single group.. and there are no tests.